src/counter.js

```diff
@@ -1,13 +1,27 @@
-export function createCounter() {
+export function createCounter(options = {}) {
+  const min = options.min ?? -Infinity;
+  const max = options.max ?? Infinity;
+  let value = options.initValue ?? 0;
```

Comment:
```suggestion
export function createCounter({ initValue = 0, min = -Infinity , max = Infinity} = {}) {
```

https://velog.io/@public_danuel/destructuring-assignment

이런식으로 기본값을 지정하는 건 어떨까요?